x-pack/plugins/security_solution/public/timelines/components/open_timeline/use_timeline_status.tsx

```diff
@@ -32,17 +31,15 @@ export const useTimelineStatus = ({
   templateTimelineType: TemplateTimelineTypeLiteralWithNull;
   templateTimelineFilter: JSX.Element[] | null;
 } => {
-  const [selectedTab, setSelectedTab] = useState<TemplateTimelineTypeLiteralWithNull>(
-    disableTemplate ? null : TemplateTimelineType.elastic
-  );
+  const [selectedTab, setSelectedTab] = useState<TemplateTimelineTypeLiteralWithNull>(null);
```

Comment:
I think we should default it to `elastic` or `custom` instead of null. I think we do not have a column in the table to show the difference between `elastic` and `custom`. I mean we won't know what kind of template it is. Maybe we can add aa call out to tell the user that they can filter.

Reply:
or we add a column to show template type `custom' vs `elastic` might be easier.